### DISCUSSION TITLE
Found 0 WI directly associated with build

### DISCUSSION THREAD

**Discussion by JGSilverman** [rfennell/AzurePipelines#952] [Q&A]
In my Azure Devops, I have a work item that is linked to a branch.

I'm using the Powershell version of Generate Release Notes.

I have this in my build definition. 

```
- task: GenerateReleaseNotes@3
  inputs:
    outputfile: '$(System.DefaultWorkingDirectory)\releasenotes.md'
    templateLocation: 'InLine'
    inlinetemplate: '# Release notes for build $defname  
      **build**: $($build)   
      
      
      ***Note** that if the is a single associated item then then size is empty, but the item is returned  
      If there are multiple associated items then the size is >0  but you need to use an index to see each item*   
      **workitems:** [Size $($workitems.length)] $($workitems)     
      **workitems:** [Size $($workitems.length)] $($workitems[0])     
      
      **changesets:**[Size $($changesets.length)] $($changesets)     
      **changesets:**[Size $($changesets.length)] $($changesets[0])  
        
      ## Associated work items (Dump of all fields)  
      @@WILOOP@@  
      * **widetail:** $($widetail)
      * **widetail.fields:** $($widetail.fields)  
      * **widetail.links:** $($widetail.fields)  
      @@WILOOP@@  
        
      ## Associated change sets/commits (Dump of all fields) 
      @@CSLOOP@@  
      * **csdetail:** $($csdetail)    
      @@CSLOOP@@'
    usedefaultcreds: true
    generateForOnlyPrimary: False
    generateForOnlyTriggerArtifact: False
```
But it's not finding any work items. See debug output.

##[debug]    Getting build details for BuildID [5825]
##[debug]Using default credentials
##[debug]        Getting up to 50 associated changesets/commits for build [5825]
##[debug]Using default credentials
##[debug]        Getting up to 50 associated work items for build [5825]
##[debug]Using default credentials
##[debug]        Found 0 WI directly associated with build
##[debug]        Running in directly associated WI only mode
##[debug]Using template mode [InLine]
##[debug]Using in-line template
##[debug]Processing template
##[debug]There are [1] builds to process
##[debug]RENDER ERROR: cannot process [***Note** that if the is a single associated item then then size is empty, but the item is returned If there are multiple associated items then the size is >0  but you need to use an index to see each item* **workitems:** [Size $($workitems.length)] $($workitems) **workitems:** [Size $($workitems.length)] $($workitems[0])]
##[debug]RENDER ERROR: cannot process [**changesets:**[Size $($changesets.length)] $($changesets) **changesets:**[Size $($changesets.length)] $($changesets[0])]
##[debug]Writing to output file [D:\_agentmb\_work\8\s\releasenotes.md].
##[debug]Skipping setting output variable name as parameter was not set.

**Comment by rfennell** [comment]
This is a duplicate of the Issue #953

**Comment by JGSilverman** [comment]
I don't understand. Issue 953 is my issue.

**Reply by rfennell** [reply]
I meant that this question was identical to #953 so I would answer it in the issue, as opposed to here
